### PR TITLE
zabbix_host : add the ability to set inventory_mode

### PR DESCRIPTION
**Pull request by ritzk**
Allows user to set inventory_mode

### CODE REVIEW

File: monitoring/zabbix_host.py [hunk 1]
```diff
@@ -356,6 +362,28 @@ def link_or_clear_template(self, host_id, template_id_list):
...
+            inventory_mode = int(-1)
+
+
+        # watch for - https://support.zabbix.com/browse/ZBX-6033
+        request_str = {'hostid': host_id, 'inventory_mode': inventory_mode}
+        try:
+            if self._module.check_mode:
+                self._module.exit_json(changed=True)
+            self._zapi.host.update(request_str)
+        except Exception, e:
```
Comment (resmo) [inline monitoring/zabbix_host.py:385]: catch all exception should be avoided.

**Comment by resmo** [comment]
would you mind squash the commits into one?

**Comment by ritzk** [comment]
Hi @resmo 
 Not sure, which all Exception I should be catching. Any guidelines would be helpful.

**Comment by resmo** [comment]
@ritzk I see there are a few catch all exceptions in this module. So one more doesn't hurt. Can be addressed later.

**Comment by gregdek** [comment]
@cove Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

**Pull request closed without merging**